tooling-mp-jwt-auth-tool/src/test/java/org/jboss/eap/qe/microprofile/jwt/auth/tool/EcKeyHelperTest.java

```diff
@@ -0,0 +1,28 @@
+package org.jboss.eap.qe.microprofile.jwt.auth.tool;
+
+import org.junit.Test;
+
+public class EcKeyHelperTest {
+
+    /**
+     * @tpTestDetails Reads a private key from a pem file
+     * @tpPassCrit Expect no Exceptions
+     * @tpSince EAP 7.4.0.CD19
```

Comment:
This was added later.

Reply:
@honza-kasik yes I added them last Friday

Reply:
@tommaso-borgato 😄 I meant in CD23...

Reply:
@honza-kasik :rofl: you r right .... fixed in all tests in the same package